### DISCUSSION TITLE
Assets not synced and nginx configuration

### DISCUSSION THREAD

**Discussion by IamSAB** [GetPublii/Publii#1901] [Help]
Hi, I sync Publii to a Docker Nginx docker container. Upload works via a shared volume using a SFTP container. I can connect and sync files but when I load the page, no CSS an Javascript loads. style.css and scripts.min.js have a 404.

So I adapted the nginx configuration to serve the assets folder as follows:
```
# Serve assets like CSS, JS, and images from /assets/
    location /assets/ {
        root /usr/share/nginx/html;
        try_files $uri $uri/ =404;
    }
```

It still could not load the mentioned files, so I checked in the nginx container for these files but they miss. Not a single assets is synced, only the folder structure.
To test if sync works at all I created a test page what works.

Is there a specific nginx configuration required or on Publii some more advanced settings i missed?

**Comment by IamSAB** [comment]
Any help? Anyone else using Docker?

**Comment by dziudek** [comment]
Hi,

The asset files exists on your server or not? If yes, then it must be an issue with domain configuration under server settings

**Reply by IamSAB** [reply]
No, the assets are not uploaded.

**Reply by dziudek** [reply]
@IamSAB - do you have any errors in the deployment process/errors logs regarding asset files?
